mongo/filterstore.go

```diff
@@ -209,11 +228,7 @@ func (s *FilterStore) CreateFilterOutput(filter *models.Filter) error {
 	session := s.Session.Copy()
 	defer session.Close()
 
-	if err := session.DB(s.db).C(s.outputsCollection).Insert(filter); err != nil {
-		return err
-	}
-
-	return nil
+	return session.DB(s.db).C(s.outputsCollection).Insert(filter)
```

Comment:
We need to create a valid mongo timestamp on insert, this can be done by the following prior to return statement

`filter.UniqueTimestamp = bson.NewMongoTimestamp(currentTime, 1)`